src/arch/x86/x86_64/src/ioport.rs

```diff
@@ -0,0 +1,25 @@
+// ioports reads and writes bytes to ioports, using the offset.
```

Comment:
This module isn't included anywhere. You could likely just dump this code inside the lib.rs file.